conformance/tests/directives_reveal_type.py

```diff
@@ -0,0 +1,24 @@
+"""
+Tests the typing.reveal_type function.
+"""
+
+# Specification: https://typing.readthedocs.io/en/latest/spec/directives.html#reveal-type
+
+from typing import Any, assert_type, reveal_type
```

Comment:
Should we also check the version where it isn't imported?

Reply:
That's not part of the spec, so I think it should be left out. It's not something that developers should rely on anyway.